website/docs/cloud-docs/api-docs/organizations.mdx

```diff
@@ -878,3 +902,4 @@ The following relationships may be present in various responses.
 | `authentication-token`| The API token for an organization.                                                                                            |
 | `entitlement-set`     | The entitlement set that determines which Terraform Cloud features the organization can use.                                  |
 | `subscription`        | The current subscription for an organization.                                                                                 |
+| `default-agent-pool`  | An agent pool that is set as the organization's default, will only be set if the `default-execution-mode` is `agent`.         |
```

Comment:
I'm not sure if this is still correct, but suggestion for active voice to clarify who sets the value:
```suggestion
| `default-agent-pool`  | An organization's default agent pool, Terraform Cloud only sets this value if the `default-execution-mode` is `agent`.         |
```

Reply:
It is the user who sets this value, so maybe we can copy the suggestions you have for the workspaces and so something like:
`Do _not_ specify this value if you set `default-execution-mode` to `remote` or `local`.`

Reply:
Building on Jarrett's suggestion, I kind of like phrasing this positively v.s. negatively, like so:
```suggestion
| `default-agent-pool`  | An organization's default agent pool. Set this value if your `default-execution-mode` is `agent`.         |
```